### DISCUSSION TITLE
Question: Closing Track while keeping data channel open.

### DISCUSSION THREAD

**Discussion by tartavull** [pion/webrtc#1820] [Q&A] [answered]
There is a Close method for DataChannel but not for TrackRemote.
I'm trying to build a client that does processing on data channels but avoids streaming in all of the video information.

Is this possible?

Thanks for Pion!

**Comment by Sean-Der** [comment, marked as answer]
Hey @tartavull thanks for using Pion!

WebRTC doesn't provide anything at the protocol to close a remote track. If you wish to inform a remote peer that you are unwilling to accept media you can reject the media during signaling.

You probably will need to implement some communication outside of WebRTC that asks the remote peer to call `RemoveTrack` and then `CreateOffer` again.
